t/25-cache.t

```diff
@@ -329,7 +330,8 @@ subtest 'cache directory is symlink' => sub {
 
     $cache->get_asset($host, {id => 922756}, 'hdd', 'sle-12-SP3-x86_64-0368-200@64bit.qcow2');
     like $cache_log, qr/Downloading "sle-12-SP3-x86_64-0368-200\@64bit.qcow2" from/, 'Asset download attempt';
-    like $cache_log, qr/Download of ".*sle-12-SP3-x86_64-0368-200.*" successful, new cache size is 1024/,
+    like $cache_log,
+      qr/Download of ".*sle-12-SP3-x86_64-0368-200.*" successful \([\d\.]+ \w+\/s\), new cache size is 1024/,
```

Comment:
I would have made it simple and used

```
      qr/Download of ".*sle-12-SP3-x86_64-0368-200.*" successful.*new cache size is 1024/,
```

as I would not care about the *exact* wording for this functional test. As long as we ensure that valid data is there once, that should be enough

Reply:
True, testing it once would have been enough.